typed-builder-macro/src/field_info.rs

```diff
@@ -73,6 +75,20 @@ impl<'a> FieldInfo<'a> {
         }
     }
 
+    pub fn setter_method_name(&self) -> Ident {
+        let name = strip_raw_ident_prefix(self.name.to_string());
+
+        if let (Some(prefix), Some(suffix)) = (&self.builder_attr.setter.prefix, &self.builder_attr.setter.suffix) {
+            Ident::new(&format!("{}{}{}", prefix, name, suffix), Span::call_site())
+        } else if let Some(prefix) = &self.builder_attr.setter.prefix {
+            Ident::new(&format!("{}{}", prefix, name), Span::call_site())
+        } else if let Some(suffix) = &self.builder_attr.setter.suffix {
+            Ident::new(&format!("{}{}", name, suffix), Span::call_site())
+        } else {
+            Ident::new_raw(&name, Span::call_site())
```

Comment:
The point is that here you should return `self.name.clone()`, so that if the field name was `r#`'d the setter method name would be `r#`'d as well.

Reply:
Oh man, sorry. Totally missed that. Will change asap.

Reply:
Changed in c38d686